### PR TITLE
fixed preventDropToInput to make it works with ExtendedTextInteraction

### PR DESCRIPTION
**Pull request by zagovorichev**

### CODE REVIEW

File: views/js/runner/plugins/tools/preventDropToInput.js [hunk 1]
```diff
@@ -55,16 +55,37 @@ define([
...
-                            event.preventDefault();
+                        .off('.preventdropimg')
+                        // Drop event doesn't work in case that ExtendedTextInteraction is included
+                        // so for the prevent dropping - just make element as disabled, the restore its state
+                        .on(namespaceHelper.namespaceAll('dragover', 'preventdropimg'), function (event) {
+                            // if img then I don't want to do drop
+                            if (dropped.tagName === 'IMG'
+                                || dropped.tagName === 'OBJECT'
+                                || dropped.tagName === 'EMBED'
+                                || dropped.tagName === 'SVG'
```
Comment (zagovorichev) [inline views/js/runner/plugins/tools/preventDropToInput.js]: I'm not sure that we need all these checks, as we're using IMG tag for SVG

File: views/js/runner/plugins/tools/preventDropToInput.js [hunk 2]
```diff
@@ -55,16 +55,37 @@ define([
...
                     $items
-                        .off('.preventdrop')
-                        .on(namespaceHelper.namespaceAll('drop dragover', 'preventdrop'), function (event) {
-                            event.preventDefault();
+                        .off('.preventdropimg')
+                        // Drop event doesn't work in case that ExtendedTextInteraction is included
+                        // so for the prevent dropping - just make element as disabled, the restore its state
+                        .on(namespaceHelper.namespaceAll('dragover', 'preventdropimg'), function (event) {
+                            // if img then I don't want to do drop
+                            if (dropped.tagName === 'IMG'
```
Comment (krampstudio) [inline views/js/runner/plugins/tools/preventDropToInput.js]: please verify `dropped` is not `null` or ` undefined`

File: views/js/runner/plugins/tools/preventDropToInput.js [hunk 3]
```diff
@@ -55,16 +55,37 @@ define([
...
+                                || dropped.tagName === 'OBJECT'
```
Comment (krampstudio) [inline views/js/runner/plugins/tools/preventDropToInput.js]: Please move all the restricted tags name in the config, so we can adapt them if needed : 
```
if( dropped && (_.contains(restrictedTags, dropped.tagName.toLowerCase()) ) {
```

File: views/js/runner/plugins/tools/preventDropToInput.js (hunk 1 above)
Comment (krampstudio) [reply views/js/runner/plugins/tools/preventDropToInput.js]: Can an inline SVG be d&d ?

Comment (zagovorichev) [reply views/js/runner/plugins/tools/preventDropToInput.js]: no, it can't. I left only `img` tag

**Comment by zagovorichev** [comment]
@krampstudio Update

**Pull request merged**